components/chatlist.js

```diff
@@ -43,15 +48,15 @@ function ChatList({ currentUser, data, isDarkTheme }) {
 
                   key={e.key} style={styles.container}>
                     <View>
-                    <Image style={styles.images} source={e.photo}/>
+                    <Image style={styles.images}  source={e.thumbnail ? e.thumbnail : require('../data/photos/tester.png')} />
                     </View>
                     <View style={isDarkTheme ? styles.borderDark : styles.border}>
-                    <Text style={isDarkTheme ? styles.usernameDark : styles.username}> {e.name}</Text>
-                    <Text style={isDarkTheme ? styles.unreadDark : styles.unread}> {e.messages.length ? e.messages.length + ' new messages' : 'no new messages'} {e.photomessages.length > 0 ?  ' 📸' : ''}  </Text>
+                    <Text style={isDarkTheme ? styles.usernameDark : styles.username}> {e.first_name}</Text>
+                    {/* <Text style={isDarkTheme ? styles.unreadDark : styles.unread}> {e.messages.length ? e.messages.length + ' new messages' : 'no new messages'} {e.photomessages.length > 0 ?  ' 📸' : ''}  </Text> */}
                     </View>
 
                   </Text>
-                })}
+                }) : null }
```

Comment:
I had not realized you could return null in React Native, but I like that as an implementation!

Reply:
If they have messages it will map and return a list of people they have messages with. But if they have no friends I'm not mapping through a empty array instead making it null or the app will crash.